internal/server/main.go

```diff
@@ -24,9 +25,16 @@ func Run(dblogPath string) {
 	bus := bus.New()
 	appDB := db.New(&runtime.Executor{})
 
+	dbLog, err := oplog.OpenGobLog(dblogPath)
+	defer dbLog.Close()
+	if err != nil {
+		panic(err)
+	}
+
 	modules := []lib.Module{
 		gzip.GetModule(),
 		cors.GetModule(),
+		audit.GetModule(bus, dbLog),
```

Comment:
the audit log, i think, is meant to be different than the DB log -- it's meant to track the lib.ParsedRequest event. In other words, tracking operations (and who performed them) vs the actual data itself.

I do think it would be cool to have a DB log viewer in catalog if that's what you're going for here, but I'd call it something different/factor it out a bit.

Reply:
Ahhh that wasn't clear to me, but that makes sense.